docs/docs/middlewares/path.md

```diff
@@ -0,0 +1,41 @@
+# Path
+This function returns the URL for the given route. 
+
+## Parameters
+```path(page, queries, locale)```
+
+parameter        | type           | required | description
+-----------------|----------------|----------|------------
+page             | string         | ✓        | The name of page (e.g. www.mysite.com).
+queries          | array          | ❌        | Array of url parameters (e.g. [gallery: nature])
```

Comment:
I think we use this as an object/map, not an array: `{{ path('story', { 'story_slug': story.slug }) }}`. But I may be wrong, terminology is not my cup of tea.